etcdserver/server.go

```diff
@@ -594,7 +594,13 @@ func (s *EtcdServer) run() {
 
 func (s *EtcdServer) applyAll(ep *etcdProgress, apply *apply) {
 	s.applySnapshot(ep, apply)
+	st := time.Now()
```

Comment:
I think this would be better in https://github.com/coreos/etcd/blob/master/etcdserver/server.go#L577